src/app/workflow/view/view.service.ts

```diff
@@ -102,25 +138,32 @@ export class ViewService {
     const dialogData: ConfirmationDialogData = {
       message: `A Digital Object Identifier (DOI) allows a version to be easily cited in publications and can't be
                 undone, though some metadata will remain editable. It can take some time to request a DOI.
-                Are you sure you'd like to create a DOI for version
+                Are you sure you'd like to request a DOI for version
                 <b>${version.name}</b>?`,
-      title: 'Issue DOI',
-      confirmationButtonText: 'Issue DOI',
+      title: 'Request DOI',
+      confirmationButtonText: 'Request DOI',
       cancelButtonText: 'Cancel'
     };
 
-    this.confirmationDialogService.openDialog(dialogData, bootstrap4mediumModalSize).subscribe(confirmationResult => {
-      if (confirmationResult) {
-        this.workflowsService
-          .requestDOIForWorkflowVersion(workflow.id, version.id)
-          .subscribe(
-            (versions: Array<WorkflowVersion>) => this.requestDOISuccess({ ...workflow, workflowVersions: versions }, version),
-            (errorResponse: HttpErrorResponse) => this.alertService.detailedError(errorResponse)
-          );
-      } else {
-        this.alertService.detailedSuccess('You cancelled DOI issuance.');
-      }
-    });
+    const workflowName: string = workflow.workflowName || workflow.repository;
+
+    this.confirmationDialogService
+      .openDialog(dialogData, bootstrap4mediumModalSize)
+      .pipe(first())
+      .subscribe(confirmationResult => {
+        if (confirmationResult) {
+          this.alertService.start(`A Digital Object Identifier (DOI) is being requested for workflow
+                                       "${workflowName}" version "${version.name}"!`);
+          this.workflowsService
+            .requestDOIForWorkflowVersion(workflow.id, version.id)
+            .subscribe(
+              (versions: Array<WorkflowVersion>) => this.requestDOISuccess({ ...workflow, workflowVersions: versions }, version),
```

Comment:
It wasn't easy to tell here that a success message is displayed, and that the start message "state" was exited. If it's not the indeterminate progress bar will go on forever. You do it in `this.requestDOISuccess`, but in the future I suggest the start/success/error invocations all be in the same method, that way it's easier to tell if it's being properly "terminated".

I don't think it's worth changing for this PR, as this code will probably be replaced with the wizard we've talked about, so just FYI,